tests/queries/0_stateless/01194_http_query_id.sh

```diff
@@ -4,14 +4,14 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-url="http://${CLICKHOUSE_HOST}:${CLICKHOUSE_PORT_HTTP}/?session_id=test_01194"
 rnd=$RANDOM
+url="${CLICKHOUSE_URL}&session_id=test_01194_$RANDOM"
```

Comment:
Btw, bash's RANDOM only has about 32768 unique values, and it means that for our 100 parallel invocations in the flaky check, it has a high chance to hit the birthday paradox.